api/bootstrap.proto

```diff
@@ -0,0 +1,27 @@
+// This proto is expected to be provided on disk or via the command-line to
+// Envoy. It provides sufficient information for Envoy to fetch the rest of
+// its configuration from either disk or management server(s).
+
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "api/base.proto";
+import "api/cds.proto";
+
+message Bootstrap {
+  // Node identity to present to the management server and for instance
+  // identification purposes (e.g. in generated headers).
+  Node node = 1;
```

Comment:
It basically implies both, the ID is there and service name can be part of the metadata. We might want to make service name first class, not sure.